docs/conf.py

```diff
@@ -152,7 +152,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-# html_title = None
+html_title = "Parcels Documentation"
```

Comment:
Would "OceanParcels" be better here?

Reply:
No, the code is called Parcels. OceanParcels is the name of the organisation that develops the Parcels code (mostly because parcels.org was already taken). I realise it's confusing...